.circleci/config.yml

```diff
@@ -300,6 +300,55 @@ jobs:
             - integration-tests/build/outputs/apk/androidTest/latestDependencies/release/integration-tests-latestDependencies-release-androidTest.apk
             - integration-tests/build/outputs/apk/androidTest/unityIAP/release/integration-tests-unityIAP-release-androidTest.apk
 
+  purchases-integration-tests-build:
+    <<: *android-executor
+    steps:
+      - checkout
+      - install-sdkman
+      - revenuecat/install-gem-unix-dependencies:
+          cache-version: v1
+      - android/restore-build-cache
+      - run:
+          name: Create purchases integration tests apks
+          command: |
+            bundle exec fastlane android build_purchases_integration_tests
+      - android/save-build-cache
+      - persist_to_workspace:
+          root: .
+          paths:
+            - purchases/artifacts/integrationTest-app.apk
+            - purchases/artifacts/integrationTest-test.apk
+
+  run-firebase-tests-purchases-integration-test:
+    description: "Run purchases module integration tests for Android in Firebase. Variant integrationTest"
+    executor: gcp-cli/google
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - gcp-cli/initialize:
+          gcloud-service-key: GCLOUD_SERVICE_KEY
+          google-compute-zone: GOOGLE_COMPUTE_ZONE
+          google-project-id: GOOGLE_PROJECT_ID
```

Comment:
I'm reusing the same firebase cloud project as the current integration tests. I actually created a new project and thought about moving them there... But I don't think there is a good reason not to simply reuse the existing one. Lmk if you have any concerns.